modules/features/profile/src/main/java/au/com/shiftyjelly/pocketcasts/profile/ProfileFragment.kt

```diff
@@ -116,6 +117,9 @@ class ProfileFragment : BaseFragment() {
             },
             modifier = Modifier.fillMaxSize(),
         )
+    }.apply {
+        // if we consume the insets on older versions of Android such as SDK 26 it will effect the layout of other areas of the app such as the What's New bottom sheet
+        consumeWindowInsets = false
```

Comment:
Until we convert everything over to Compose we might need to add this to ComposeView in fragments.
```
    override fun onCreateView(
        inflater: LayoutInflater,
        container: ViewGroup?,
        savedInstanceState: Bundle?,
    ) = content {
    ...
    }.apply {
      consumeWindowInsets = false
    }
```

Reply:
That's a great idea. I thought it might get overlooked if the property was set at the bottom of the method when examining why one page worked and another didn't. I will add this in a future PR.